### ISSUE TITLE
Documentation about required connections is incomplete

### ISSUE THREAD

**Issue by schaarsc** [nextcloud/documentation#5285]
### Steps to reproduce
https://docs.nextcloud.com/server/20/admin_manual/installation/harden_server.html#connections-to-remote-servers

### Expected behaviour
documentation should also mention connections required for featured apps.
for example
- apps are downloaded from github not from apps.nextcloud.com (github is using cloud storage for some assets)
- weather is connecting to ...?

**Comment by kesselb** [comment]
> weather is connecting to

https://api.met.no/weatherapi

**Comment by MorrisJobke** [comment]
> * apps are downloaded from github not from apps.nextcloud.com (github is using cloud storage for some assets)

We can't control this. The appstore basically is just a collection of URLs where an app can be downloaded. It could be any website out there. Of course most of the times app maintainers use GitHub releases, but there are also apps that are hosted on private servers or on gitlab or such services. So maybe a proper explanation is better than the "this is a full list of URLs/IPs".

**Comment by tflidd** [comment]
Yes and the same for apps themselves, that they can use external connections (weather, news, radio, mail, ...)

**Comment by schaarsc** [comment]
this issue is partly solved by Commit 6acedce, which added more information about the app store.

About hosts used by apps: would it be possible to add a section in the "app metadata" with required connections? This would allow to compile a list of hosts from the metadata of installed apps. (if the maintainer of the app includes this information in the metadata). 

I recently discovered that nextcloud is accessing
* garm2.nextcloud.com
* garm3.nextcloud.com

those hosts are missing in the documentation
